### PR TITLE
UI for all pages

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Made U for the pages with all functionalities.
Please review my PR thanks
Live-https://piston-notes-maker-4c8bohfbt-dev-akshatgupta.vercel.app/

### CODE REVIEW

**Review by nayyyhaa — APPROVED** [review]
redux slices are well written, code is also readable. Nice work! One suggestion tho, try to make PRs short.

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,92 @@
...
+    dispatch(getAllTrash());
+  }, []);
+  const location = useLocation();
   return (
     <div className="App">
-      <AuthenticationPage />
+      <AllRoutes />
       <ToastContainer />
+      {modalDisplay && (
+        <AddNoteModal
```
Comment (mikesch803) [inline src/App.js:35]: I tried to make notes on other pages, but no note has been created. check this

File: src/Components/Note/Note.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <div>
+            <h4
+              className={`focus:outline-none text-gray-800 dark:text-gray-100 font-bold mb-3 `}
+            >
+              {noteObj.title}
+            </h4>
+            <div
+              className={`focus:outline-none text-gray-800 dark:text-gray-100 text-sm ${noteObj.color.color} break-normal`}
+              dangerouslySetInnerHTML={{
+                __html: DOMPurify.sanitize(noteObj?.content),
```
Comment (mikesch803) [inline src/Components/Note/Note.jsx:30]: this is new to me, what is this ?

**Review by mikesch803 — APPROVED** [review]
Good to merge, the color plallate is not working, else everything is good

File: src/Components/Note/Note.jsx (hunk 2 above)
Comment (Dev-AkshatGupta) [reply src/Components/Note/Note.jsx:30]: Hey thanks for reviewing my pr man . This is just for saving our website from any kind of front end manipulations

**Pull request merged**